bricks/ai/src/chat-view/components/MessageItem/Markdown/CodeBlock/CustomComponent.tsx

```diff
@@ -62,9 +65,17 @@ export default function CustomComponent({
   const commandBrickConf = useMemo(() => {
     const config = commandBricks?.[language];
     if (config) {
+      const data = {
+        data: parseData,
+        agentId,
+        conversationId,
+        taskId,
+      };
       return {
-        component: (
-          <ReactUseBrick useBrick={config.useBrick} data={parseData} />
+        component: Array.isArray(config.useBrick) ? (
+          <ReactUseMultipleBricks useBrick={config.useBrick} data={data} />
+        ) : (
+          <ReactUseBrick useBrick={config.useBrick} data={data} />
```

Comment:
ReactUseMultipleBricks 支持单个或数组 useBrick

```suggestion
        component: <ReactUseMultipleBricks useBrick={config.useBrick} data={data} />
```